frontend/src/scenes/pipeline/transformationsLogic.tsx

```diff
@@ -0,0 +1,199 @@
+import { actions, afterMount, connect, kea, listeners, path, reducers, selectors } from 'kea'
+import { loaders } from 'kea-loaders'
+import api from 'lib/api'
+import { PluginConfigTypeNew, PluginType } from '~/types'
+import posthog from 'posthog-js'
+
+import type { pipelineTransformationsLogicType } from './transformationsLogicType'
+import { teamLogic } from 'scenes/teamLogic'
+import { canConfigurePlugins } from 'scenes/plugins/access'
+import { userLogic } from 'scenes/userLogic'
+
+function capturePluginEvent(event: string, plugin: PluginType, pluginConfig: PluginConfigTypeNew): void {
+    posthog.capture(event, {
+        plugin_id: plugin.id,
+        plugin_name: plugin.name,
+        plugin_config_id: pluginConfig.id,
+    })
+}
+
+export const pipelineTransformationsLogic = kea<pipelineTransformationsLogicType>([
+    path(['scenes', 'pipeline', 'pipelineTransformationsLogic']),
```

Comment:
The convention is that this path matches the relative file location. Should the file be called `pipelineTransformationsLogic`?